nconf/bugsnag.go

```diff
@@ -8,28 +8,28 @@ import (
 
 type BugSnagConfig struct {
 	Environment    string
```

Comment:
both yaml & json support case insensitive unmarshalling. You only need the yaml/json tags when you're (1) outputting to a case sensitive format (2) using something like snake case. 

https://play.golang.org/p/Q4BN7N3bnZZ

Reply:
according to that playground the yaml examples didn't work though?

Reply:
Wow. I read right past that. I definitely was like "yes that works". 

It turns out that the yaml lib actually automatically downcases _every key_. https://github.com/go-yaml/yaml/issues/123

Meaning for yaml, I'd have to actually make it `yaml:"Environment"`, which would break the lowercase version.